src/components/common/icons/Facebook.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Facebook = () => {
+  return (
```

Comment:
```suggestion
const Facebook = () => (
```
You can implicitly return in these cases

Reply:
Looks way cleaner thanks, I've changed it in all the components that wont have logic.